handbook/engineering/distribution/goals.md

```diff
@@ -2,78 +2,91 @@
+  - CE/CS can relay site-admins feedback and requirements which we can report on.
 - **Milestones**:
-  - ~~Research and evaluate possible tools.~~ Done: `3.19`
-    - ~~Bash/JQ.~~
-    - ~~Cue.~~ Done: `3.17`
-    - ~~Dhall.~~ Done: `3.18`
-  - ~~Create a Kubernetes deployment PoC.~~ Done: `3.19`
-  - ~~Design Dhall Architecture.~~ Done: `3.20`
-  - ~~Create a demo video/presentation to gather customer feedback.~~ Done: `Dist: 2020.11.30`
-  - Migrate internal deployments to Dhall, onboard other teams on how to make changes with Dhall and collect feedback. **In progress**
-  - Create customer facing migration path and how-to documentation. _Estimated: FY21-Q1_
-  - Use Dhall to generate public Kubernetes manifests. _Estimated: FY21-Q1_
-  - Migrate a highly technical customer to Dhall and collect usage feedback. _Estimated: FY21-Q1_
-  - Design customization workflow. _Estimated: FY21-Q1_
-    - Potentially Kustomize would still be used for last-mile changes and non-standard derivations.
-
-### [Split Cloud infrastructure into separate GCP projects](https://github.com/orgs/sourcegraph/projects/92)
-
-- **Owner**: Gonza
-- **Status**: In progress. Estimated complete by end of 2020.
+  - We have defined an initial set of data points we wish to collect.
+  - The is a process and tool in place that allows CE/CS and direct site-admins to relay feedback provided by site-admins.
+  - We have collected data from 10 site-admins.
+
+### [Improve internal deployment pipeline UX](https://github.com/orgs/sourcegraph/projects/96)
+
+Our existing deployment pipelines to our Sourcegraph instances (such as Sourcegraph Cloud) has several usability problems - for example, it is hard for engineers to identify when a commit was deployed to an environment or which deployment is currently running in a particular environment. We want to improve the deployment experience, making sure we can deploy with confidence and can easily understand in which stage of the pipeline a change currently is.
+
+- **Status**: Not started. _Estimated: FY22-Q1_
 - **Outcomes**:
-  - We have a clear guideline on when to split resources to new projects.
-  - We report the cost of each project.
-  - We can set spending limits for dynamic environments.
+  - ~~We can correlate deployments to Grafana metrics.~~ _Done_
+  - Engineers can easily identify on which stage of our full CI/CD pipeline is a given commit.
+  - Engineers can determine what commits are currently running on a given environment.
+  - Engineers are optionally notified when their commits are deployed.
+  - Engineers can trigger a rollback to a previous version.
+  - Stretch: Rollbacks stop the line and prevent any further deployments.
+  - TBD: Branches can trigger a parallel `sourcegraph-frontend` deployments.
+  - TBD: We can trigger rollbacks and deployments via a `/` command in Slack.
+  - TBD: Migrations and other dangerous operations have an automated method of being performed.
```

Comment:
```suggestion
```
I believe this requires and is tracked in https://github.com/sourcegraph/sourcegraph/issues/16909